official/resnet/keras/keras_cifar_main.py

```diff
@@ -132,16 +132,16 @@ def run(flags_obj):
       num_epochs=flags_obj.train_epochs,
       parse_record_fn=parse_record_keras)
 
-  optimizer = keras_common.get_optimizer()
   strategy = distribution_utils.get_distribution_strategy(
```

Comment:
what happens when `strategy` is `None`? Which is possible right?

Reply:
Good catch. Fixed.